user_guide_src/source/models/model.rst

```diff
@@ -562,37 +562,6 @@ value an array of fieldnames of interest.::
     // get the rules for only the "city" and "state" fields
     $rules = $model->getValidationRules(['only' => ['city', 'state']]);
 
-Validation Placeholders
------------------------
```

Comment:
wait, are you sure about removing `Validation Placeholders` documentation from Model ? It seems still valid?

Reply:
@jreklund I tried your changes, and `validationRules` definition in Model still working, please revert this removal.

Reply:
I did, the code are only in the Model due to BC. I moved it into the Validation part of the user guide as it's not unique to the Model any more.

It will work from Models, Controller, Libraries etc. Every time you execute $validation->run(), so it's not bound to Model.

Reply:
Yes, but that's still work if we define `validationRules` in the model, I think the _documentation_ of defining `validationRules` needs to be kept as it will still works if we define there.